sysutils/fastfetch/Portfile

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem       1.0
+PortGroup        github 1.0
+PortGroup        cmake 1.0
+
+github.setup     fastfetch-cli fastfetch 2.5.0
+platforms        any
+supported_archs  noarch
+license          MIT
+categories       sysutils
+maintainers      {}
```

Comment:
this should be "nomaintainer" if there is no maintainer. Or you could volunteer to maintain it and then you add your GitHub handle and obfuscated e-mail within the curly brackets. Ideally you would also add "openmaintainer".